packages/first-boot-wizard/files/bin/firstbootwizard

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/lua
+
+local fbw = require('firstbootwizard')
+
+print("[FBW] Scanning...")
+get_all_networks()
```

Comment:
it doesn't seem that this command is useful for anything... is it?
if the get_all_networks stores the info somewhere for later use, should change the signature of the method.
Also, would add a print("[FBW] Scanning done") sentence

Reply:
The get_all_networks method prints that information and more.